components/nexus-repository/src/main/java/org/sonatype/nexus/repository/view/ContentTypes.java

```diff
@@ -19,15 +19,28 @@
  */
 public class ContentTypes
 {
+  /**
+   * Plain text.
+   */
   public static final String TEXT_PLAIN = "text/plain";
 
+  /**
+   * Generic HTML content type.
+   */
   public static final String TEXT_HTML = "text/html";
 
-  public static final String TEXT_XML = "text/xml";
```

Comment:
The other difference between application/xml and text/xml is wrt. clients that can't handle the mime-type. With application/xml the client is likely to prompt you to save it as a file, whereas with text/xml the client will attempt to render it as plain text. That said... there was talk a while back of deprecating text/xml: http://tools.ietf.org/html/draft-murata-kohn-lilley-xml-03#section-3.1 so I'm not that worried about removing this constant. Otherwise we could replace it with a short comment pointing people to application/xml... or leave it there but marked as `@Deprecated`?